src/codegen/mod.rs

```diff
@@ -1216,7 +1212,9 @@ impl Bitfield {
         let bitfield_ty_layout = bitfield_ty
             .layout(ctx)
             .expect("Bitfield without layout? Gah!");
-        let bitfield_int_ty = helpers::blob(ctx, bitfield_ty_layout);
```

Comment:
Hmm, so was the bitfield layout really misaligned? How does `helpers::blob` come up with `[u32; 2]` rather than `u64`? I think that's the thing to fix.

Reply:
It's not misaligned.  Testing in C++ on 32 bit, I get 4 byte alignment for the bitfield and 8 byte alignment for `uint64_t`.  So it's correct in returning `[u32; 2]` for a correctly aligned 64 bit value.  It's just that we don't need to care about alignment for the argument value we're passing to `BindgenBitfieldUnit::set`.

Reply:
I see, that's a nice distinction that we should probably have a test for. I'll write one before merging if you don't get to it :)